app/code/Magento/Backend/Ui/Component/Control/SaveSplitButton.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Backend\Ui\Component\Control;
+
+use Magento\Framework\View\Element\UiComponent\Control\ButtonProviderInterface;
+use Magento\Ui\Component\Control\Container;
+
+/**
+ * @api
+ */
+class SaveSplitButton implements ButtonProviderInterface
+{
+    /**
+     * @var string
+     */
+    private $saveTarget;
+
+    /**
+     * @param string $saveTarget
+     */
+    public function __construct($saveTarget)
+    {
+        $this->saveTarget = $saveTarget;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getButtonData()
+    {
+        return [
+            'label' => __('Save &amp; Continue'),
```

Comment:
Should we really use HTML entities here? Compare to https://github.com/magento/magento2/blob/develop/app/code/Magento/Catalog/Block/Adminhtml/Product/Edit/Button/Save.php#L56.

Reply:
I use `&amp` due to reserved characters in HTML must be replaced with character entities

Reply:
Am I understand you correct that mentioned code in `Magento_Catalog` is incorrect? Or sometimes we just have two version (with `&` and with `&amp;`) and it's fine?

Reply:
Looks like is mistake in `Magento_Catalog`